code/drasil-docLang/lib/Drasil/DocumentLanguage.hs

```diff
@@ -362,10 +362,10 @@ mkSolChSpec si (SCSProg l) =
     map (mkSubSCS si) l
   where
     mkSubSCS :: SystemInformation -> SCSSub -> Section
-    mkSubSCS _ (TMs _ _ [])      = error "There are no Theoretical Models"
+    mkSubSCS _ (TMs _ _ [])      = SSD.thModF Doc.section_ []
     mkSubSCS _ (GDs _ _ [] _)    = SSD.genDefnF []
-    mkSubSCS _ (DDs _ _ [] _) = error "There are no Data Definitions"
-    mkSubSCS _ (IMs _ _ [] _)    = error "There are no Instance Models"
+    mkSubSCS _ (DDs _ _ [] _)    = SSD.dataDefnF EmptyS []
+    mkSubSCS _ (IMs _ _ [] _)    = SSD.inModelF SSD.pdStub SSD.ddStub SSD.tmStub SSD.gdStub []
```

Comment:
Are these cases covered by the cases for non-empty lists below? For example,
```haskell
mkSubSCS _ (IMs _ _ [] _) = SSD.inModelF SSD.pdStub SSD.ddStub SSD.tmStub SSD.gdStub []
```
seems to just be a specific case of
```haskell
mkSubSCS si' (IMs intro fields ims ShowDerivation) =
      SSD.inModelF SSD.pdStub SSD.ddStub SSD.tmStub SSD.gdStub $ map mkParagraph intro ++
      concatMap (\x -> [LlC $ instanceModel fields si' x, derivation x]) ims
```
If so, we should remove the first case to improve readability (the cases were explicitly separated before since they had different functionality, but now that they have similar functionality, this is likely unnecessary).

Reply:
Yes, you are correct! The second case does cover the functionality of the first case. I will remove that now!